lib/services/cordova-migration-service.ts

```diff
@@ -214,6 +214,10 @@ export class CordovaMigrationService implements ICordovaMigrationService {
 				this.$logger.trace("Migrated core plugins to: ", helpers.formatListOfNames(newPluginsList, "and"));
 				this.$project.setProperty("CorePlugins", newPluginsList, configuration);
 			});
+			
+			if(semver.gte(newVersion, "4.0.0")) {
+				this.$logger.out("Cordova versions greather than 4.0.0 support WKWebView for iOS and Crosswalk WebView for android. If you would like to change default web view you can use $ appbuilder web-view set command");
```

Comment:
`android` should be capitilized. Also consider printing the `4.0.0` version using the constant.